src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -11,6 +11,10 @@
 using System.Security;
 using System.Text;
 
+#if CoreCLR
+using System.Runtime.InteropServices;
```

Comment:
Is this used anywhere else? If not, maybe get rid of this ifdef and use the full name space below?

Reply:
Only used in the one line, but twice. We would need the full namespace on both `RuntimeInformation.IsOSPlatform()` and `OSPlatform.Windows`.

I just tried it, and I think the import is less cumbersome.